Cargo.toml

```diff
@@ -34,6 +34,7 @@ default-members = [
   "src/storage",
   "src/wkt",
   "tools/check-copyright",
+  "tools/minimal-version-helper",
```

Comment:
This does not need to be in the `default-members` list (for that matter, neither does `check-copyright`).

Reply:
I'll file an issue follow up, the commands we use in CI will need to be updated when we fix this. #3839